cmd/feed-dns/main.go

```diff
@@ -81,7 +87,13 @@ func main() {
 		log.Fatal("Unable to create k8s client: ", err)
 	}
 
-	dnsUpdater := dns.New(r53HostedZone, elbRegion, elbLabelValue, albNames, awsAPIRetries)
+	var dnsUpdater controller.Updater
+	if internalCNAME != "" && externalCNAME != "" {
+		addressesWithScheme := map[string]string{"internal": internalCNAME, "internet-facing": externalCNAME}
+		dnsUpdater = dns.New(r53HostedZone, elbRegion, addressesWithScheme, "", albNames, awsAPIRetries)
+	} else {
+		dnsUpdater = dns.New(r53HostedZone, elbRegion, nil, elbLabelValue, albNames, awsAPIRetries)
+	}
```

Comment:
Now that we have optional fields, we should probably add a config struct to pass to New.